commitlint.config.js

```diff
@@ -0,0 +1,25 @@
+module.exports = {
+  extends: ['@commitlint/config-conventional'],
+
+  helpUrl: 'https://open-edx-proposals.readthedocs.io/en/latest/oep-0051-bp-conventional-commits.html',
+
+  rules: {
+    'body-leading-blank': [0, "always"],
+
+    'subject-case': [0, "always", []],
+
+    'subject-full-stop': [0, "never", '.'],
+
+    'type-enum':
+      [2, 'always', [
+        'revert', 'feat', 'fix', 'perf', 'docs', 'test', 'build', 'refactor', 'style', 'chore', 'temp'
+      ]],
+  },
+
+  ignores: [
+    // Allow GitHub revert messages, like:
+    //    Revert "introduce a bug"
+    //    Revert "introduce a bug" (#1234)
+    message => /^Revert ".*"( \(#\d+\))?/.test(message)
```

Comment:
Sometimes people merge PRs into their PRs in which case there would be merge commits in a PR, do we want to also ignore that or ask people to update those commit messages?

Reply:
commitlint has default ignores, though the specifics are not documented.  They include merge messages: https://github.com/conventional-changelog/commitlint/blob/master/%40commitlint/is-ignored/src/defaults.ts